rustler_sys/src/rustler_sys_api.rs

```diff
@@ -138,13 +138,19 @@ pub type ErlNifResourceDown = unsafe extern "C" fn(
     mon: *const ErlNifMonitor,
 ) -> ();
 
+/// See [ErlNifResourceDynCall](http://www.erlang.org/doc/man/erl_nif.html#ErlNifResourceDynCall) in the Erlang Docs.
+pub type ErlNifResourceDynCall =
+    unsafe extern "C" fn(env: *mut ErlNifEnv, obj: *mut c_void, call_data: *const c_void) -> ();
+
 /// See [ErlNifResourceTypeInit](http://www.erlang.org/doc/man/erl_nif.html#ErlNifResourceTypeInit) in the Erlang docs.
 #[derive(Debug, Copy, Clone)]
 #[repr(C)]
 pub struct ErlNifResourceTypeInit {
     dtor: *const ErlNifResourceDtor,
     stop: *const ErlNifResourceStop, // at ERL_NIF_SELECT_STOP event
     down: *const ErlNifResourceDown, // enif_monitor_process
+    members: c_int,
```

Comment:
From the [documentation](https://erlang.org/doc/man/erl_nif.html#enif_init_resource_type):

> The struct also contains the field `members` that must be set to the number of initialized callbacks counted from the top of the struct. For example, to initialize all callbacks including dyncall, members should be set to 4. All callbacks are optional and may be set to NULL. 

I am unsure where we initialize `members`. Is this set to zero in `initmacro.rs`? (I also wonder why `members` is actually needed, if the function pointers can be `NULL`, but well..)

Reply:
We don't use `ErlNifResourceTypeInit` in `rustler` as far as I know, so this is only relevant for users that use `rustler_sys` directly.

Reply:
Ah, that makes sense! Thanks for the clarification.